src/App.js

```diff
@@ -234,7 +239,34 @@ class _AppStackWrapper extends React.Component {
     },
   });
 
-  componentDidMount = () => {
+  /**
+   * This method set the listener for the notifee foreground event
+   */
+  setNotifeeForegroundListener = () => {
+    try {
+      notifee.onForegroundEvent(async ({ type, detail }) => {
+        switch (type) {
+          case EventType.PRESS:
+            try {
+              // eslint-disable-next-line max-len
+              if (detail.pressAction && detail.pressAction.id === PUSH_TRANSACTION_ID.NEW_TRANSACTION) {
+                const txDetails = await getTxDetails(this.props.wallet, detail.notification.data.txId);
+                this.props.loadTxDetails(txDetails);
+              }
+            } catch (error) {
+              console.error('Error processing notification press event.', error);
+            }
+            break;
+          default:
+            // to nothing
+        }
+      });
+    } catch (error) {
```

Comment:
The same error handling problem here

Reply:
[8ecf541](https://github.com/HathorNetwork/hathor-wallet-mobile/pull/233/commits/8ecf54137e11c420b3faa0b779891c11e8d55d68)